app/views.py

```diff
@@ -0,0 +1,102 @@
+from flask import current_app, Blueprint, render_template, abort, request, url_for, flash, redirect
+from jinja2 import TemplateNotFound
+from .forms import AddForm
+from sqlalchemy import create_engine, Column, Integer, String, func
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+main_page = Blueprint('mainpage ', __name__)

```

Comment:
main_page is not a best name. Please come up with the better name more suitable for the blueprint